docs/assistant/security-assistant.asciidoc

```diff
@@ -8,18 +8,16 @@
 
 The Elastic AI Assistant utilizes generative AI to bolster your cybersecurity operations team. It allows users to interact with {elastic-sec} for tasks such as alert investigation, incident response, and query generation or conversion using natural language and much more.
 
-AI Assistant relies on generative AI connectors to communicate with third-party AI providers.
+AI Assistant can connect to multiple LLM providers so you can select the best model for your needs.
 
 [role="screenshot"]
-image::images/assistant.gif[Animation of AI Assistant chat window,90%]
+image::images/assistant-basic-view.png[Image of AI Assistant chat window,90%]
 
 IMPORTANT: The Elastic AI Assistant is designed to enhance your analysis with smart dialogues. Its capabilities are still developing. Users should exercise caution as the quality of its responses might vary. Your insights and feedback will help us improve this feature. Always cross-verify AI-generated advice for accuracy.
 
-[TIP]
+[Recommended models]
 ====
-When choosing a third-party provider to use with AI Assistant, remember that different services may impose rate limits on their APIs. This may negatively affect AI Assistant performance. In addition, different models support various context lengths. Models with larger context lengths will provide a better experience when using the AI Assistant.
-
-For example, refer to OpenAI's documentation on https://platform.openai.com/docs/guides/rate-limits/[rate limits] and https://help.openai.com/en/articles/7127966-what-is-the-difference-between-the-gpt-4-models[GPT-4 models] for more information on their available options.
+While AI Assistant is compatible with many different models, our testing found increased quality with Azure 32k, and faster and more cost-effective responses with Claude 3 Haiku and OpenAI GPT4 Turbo.
```

Comment:
This renders as regular paragraph text in the preview (rather than a callout or a separate section) – just checking that's intended?